### PR TITLE
Add file type client

### PR DESCRIPTION
**Pull request by hubjac1**

### CODE REVIEW

File: packages/ozone-typescript-client/src/filetypeClient/filetypeClientImpl.ts [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+
+	}
+
+	async delete(id: UUID): Promise<UUID | null> {
+		const request = new Request(`${this.baseUrl}/rest/v3/filetype/${id}`)
+			.setMethod('DELETE')
+		return this.client.call<UUID>(request).catch(returnNullOn404)
+	}
+
+	@once
```
Comment (lmancel) [inline packages/ozone-typescript-client/src/filetypeClient/filetypeClientImpl.ts]: Just to know, are we supposed to call `refreshCache` manually or is it automatically done after some time ?

File: packages/ozone-api/ozone-api-edit-video/src/ozone-api-edit-video.ts [hunk 2]
```diff
@@ -160,14 +145,10 @@ export class OzoneApiEditVideo {
...
-		const url = `${getDefaultClient().config.ozoneURL}/rest/v3/filetype/identifier/${identifier}`
-		// No senses to do this work by chance on flowr
-		// should be the format of the video under edition
-
-		const request = new OzoneAPIRequest()
-		request.url = url
-
-		return ((await request.sendRequest()).response) as OzoneType.FileType
+		const fileTypeCache = await getDefaultClient().fileTypeClient().getFileTypeCache()
+		const file = fileTypeCache.findByIdentifier(identifier)
```
Comment (lmancel) [inline packages/ozone-api/ozone-api-edit-video/src/ozone-api-edit-video.ts]: shouldn't this be `await`ed ?

File: packages/ozone-typescript-client/src/filetypeClient/filetypeClientImpl.ts (hunk 1 above)
Comment (hubjac1) [reply packages/ozone-typescript-client/src/filetypeClient/filetypeClientImpl.ts]: No for the moment it's all manual as in cache type

File: packages/ozone-api/ozone-api-edit-video/src/ozone-api-edit-video.ts (hunk 2 above)
Comment (hubjac1) [reply packages/ozone-api/ozone-api-edit-video/src/ozone-api-edit-video.ts]: no FileTypeCache does not return promises

**Pull request closed without merging**